### DISCUSSION TITLE
Load multiple control vectors at the same time

### DISCUSSION THREAD

**Discussion by HatsuneMikuUwU33** [ggml-org/llama.cpp#8078] [Ideas]
Most control vectors downscale or upscale a single trait. Loading of multiple at once will make them more useful.

**Comment by ngxson** [comment]
Unless I'm missing something, this is a feature already exist since the beginning: https://github.com/ggerganov/llama.cpp/pull/5970

![image](https://github.com/ggerganov/llama.cpp/assets/7702203/8f02c962-99e6-4c20-bce2-8bb7655a5be4)

**Comment by HatsuneMikuUwU33** [comment]
Wait what? Why does it work now? I swear when I tested it 2 days ago it gave me some kind of error. Sleep deprivation is a bitch, I guess. Thanks for showing me that it can work when typed out right. I feel so stupid right now.
